tools/internal_ci/linux/grpc_bazel_build_in_docker.sh

```diff
@@ -27,20 +27,10 @@ cd /var/local/git/grpc
 
 bazel build :all //test/... //examples/...
 
-BUILD_WITH_XDS_SIZE=$(stat -c %s "bazel-bin/test/cpp/end2end/end2end_test")
 # TODO(jtattersmusch): Adding a build here for --define=grpc_no_xds is not ideal
 # and we should find a better place for this. Refer
 # https://github.com/grpc/grpc/pull/24536#pullrequestreview-517466531 for more
 # details.
-# Test that builds with --define=grpc_no_xds=true work.
```

Comment:
Looks like this test got deleted too. Let's revert line 35 and 36

Reply:
Oh that's right. Two lines are being reverted.